### PR TITLE
Rename/docs

### PR DESCRIPTION
**Pull request by kzidane**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 DESCRIPTION = CS50 Library for Java
 MAINTAINER = CS50 <sysadmins@cs50.harvard.edu>
-NAME = lib50-java
+OLD_NAME = lib50-java
+NAME = libcs50-java
```
Comment (dmalan) [inline Makefile:4]: Did we deploy this to CS50 IDE already? (I.e., indeed necessary to have `OLD_NAME`?)

File: Makefile [hunk 2]
```diff
@@ -40,6 +41,9 @@ deb: build
 	-s dir \
 	-t deb \
 	-v "$(VERSION)" \
+	--conflicts $(OLD_NAME) \
+	--provides $(OLD_NAME) \
```
Comment (dmalan) [inline Makefile:45]: This def shouldn't be `NAME` now?

File: docs/allclasses-frame.html [hunk 3]
```diff
@@ -1,18 +0,0 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```
Comment (dmalan) [inline docs/allclasses-frame.html]: Sorry for making the mess; should've just put on `gh-pages` manually earlier. Want to purge these from the history as unnecessary?

**Review by dmalan — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (kzidane) [reply Makefile:4]: Yes, lib50-java v1.1.1 is currently installed on the IDE.

File: Makefile (hunk 2 above)
Comment (kzidane) [reply Makefile:45]: Nope. This is basically to have `libcs50-java` replace `lib50-java` per https://www.debian.org/doc/debian-policy/ch-relationships.html#s-conflicts.

> A special exception is made for _packages which declare a conflict_ with their own package name, or _with a virtual package which they provide_ (see below): this does not prevent their installation, and _allows a package to conflict with others providing a replacement for it_. You use this feature when you want the package in question to be the only package providing some feature.

Comment (dmalan) [reply Makefile:45]: 👍

**Comment by dmalan** [comment]
:shipit:

**Pull request merged**